src/github.com/cloudfoundry-incubator/webhooks/pkg/webhooks/manager/main.go

```diff
@@ -0,0 +1,56 @@
+package main
```

Comment:
Little confused here- would we want to clutter our ephemeral store with these packages and files? Instead, won't it be better to generate the Go executable via compilation or pre-release?
Or has this already been discussed?

Reply:
Webhook job is added for reference. Currently we are not using it. So can be ignored. :)

Reply:
Ok... so, you would remove the webhook job from the packages. fair enough- will approve